eng/versioning/external_dependencies.txt

```diff
@@ -67,8 +67,8 @@ io.netty:netty-tcnative-boringssl-static;2.0.62.Final
 io.netty:netty-transport-native-epoll;4.1.100.Final
 io.netty:netty-transport-native-unix-common;4.1.100.Final
 io.netty:netty-transport-native-kqueue;4.1.100.Final
```

Comment:
Also upgrade the Netty dependencies to `4.1.101.Final` to align with the changes to the Reactor Netty dependencies.